corehq/apps/reports/templates/reports/filters/explorer_columns.html

```diff
@@ -2,57 +2,62 @@
 {% load hq_shared_tags %}
 {% load i18n %}
 {% block filter_content %}
-    <div class="panel-group" id="case-list-explorer-columns" role="tablist" aria-multiselectable="true">
+    <div class="panel-group report-filter-case-property-columns"
+         id="case-list-explorer-columns"
+         role="tablist"
+         data-initialvalue="{{ initial_value }}"
+         data-columnsuggestions="{{ column_suggestions }}"
+    >
         <div class="panel panel-default">
             <div class="panel-heading" role="tab">
-                <a role="button" data-toggle="collapse" data-parent="#case-list-explorer-columns" href="#columns-editor">
-                    <h4 class="panel-title">
-                        {% trans "Edit Columns" %}
-                        <span class="pull-right" id="panel-chevron">
+                <h4 class="panel-title">
+                    <a role="button" data-toggle="collapse" data-parent="#case-list-explorer-columns" href="#columns-editor">
+                        <span id="panel-chevron">
                             <i class="fa fa-chevron-down"></i>
                         </span>
-                    </h4>
-                </a>
+                        {% trans "Edit Columns" %}
+                        <small>(</small><span data-bind="foreach: properties"><!--ko if: $index() != 0 && name() !=='' -->, <!--/ko--><small data-bind="text: name"></small></span><small>)</small>
```

Comment:
Why would a property's name be blank?

Reply:
when you click the `+ Add column` button, the new property gets created without a name, so it adds an awkward comma at the end of the list.